.github/workflows/tests.yml

```diff
@@ -40,14 +40,6 @@ jobs:
           extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick, fileinfo
           coverage: none
 
-      - name: Set PHP 8 Mockery
```

Comment:
I've removed this workflow step in favour of requiring Mockery directly in the package's `composer.json` file. If you'd prefer to keep it here, feel free to revert `0101ca47`.